presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -447,6 +447,51 @@ Property Name                                        Description
                                                      defaults to ``20``.
 ==================================================== ============================================================
 
+Performance Tuning Configuration Properties
```

Comment:
Can you move this above, to be directly after the "Hive Configuration Properties" section? Or move it below, so that it's after the Google Cloud Storage section?

Reply:
Yes, and opening a new PR in Trino. Will close this one.